staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/envelope_test.go

```diff
@@ -201,3 +202,70 @@ func benchmarkRead(b *testing.B, transformer value.Transformer, valueLength int)
 	}
 	b.StopTimer()
 }
+
+// remove after 1.13
```

Comment:
This test is probably useful indefinitely to make sure the current implementation matches a known storage format, no?